records/2021-08-16.scalability.move-api-to-lambda.md

```diff
@@ -32,7 +32,15 @@ Another consideration will be the effects on other parts of Notify. If the API c
 
 ## Load testing results
 
+We ran a load test where we simulated 5,500 requests per minute to the api using locust, to see if the new lambda deployment could meet existing peak loads. The these requests were made to the `/v2/notifications/email` endpoint and were split into 2 services to make the test more realistic. Here are the reports:
+- [Report for Service 1](attachments/report_1629229241.7761497.html)
+- [Report for Service 2](attachments/report_1629229258.5504797.html)
 
+There were 0 failures, and median response time was 110ms for both services and 99th percentile response time was 250ms for Service 1 and 230ms for Service 2 over a 7 minute test.
```

Comment:
Do you have these numbers for the k8 version so they can be compared?

Reply:
No, but I will get them!

Reply:
Done.